hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/TestRootedOzoneFileSystem.java

```diff
@@ -1225,6 +1236,15 @@ public void testTrash() throws Exception {
       }
     }, 1000, 180000);
 
+    Assert.assertTrue(getOMMetrics()
+        .getNumTrashDeletes() >= prevNumTrashDeletes);
+    Assert.assertTrue(getOMMetrics()
+        .getNumTrashRenames() > prevNumTrashRenames);
+    Assert.assertTrue(getOMMetrics()
+        .getNumTrashFilesDeletes() >= prevNumTrashFileDeletes);
```

Comment:
If for `prevNumTrashDeletes` and `prevNumTrashFileDeletes ` the equality hold true, That would mean the metrics didn't increment, which won't serve the purpose of the test. Can we ditch the `=` in the assertion and confirm the metric value gets incremented. Something like -
```
    GenericTestUtils.waitFor(
        () -> getOMMetrics().getNumTrashDeletes() > prevNumTrashDeletes
            && getOMMetrics().getNumTrashFilesDeletes()
            > prevNumTrashFileDeletes, 100, 180000);
```
Unless there is some catch here, in that case it should be ok..

Reply:
Thanks, Excellent comment. Incorporated it.